test/integration/helpers.js

```diff
@@ -63,7 +56,24 @@ var waitForUri = function(uri) {
 };
 exports.waitForUri = waitForUri;
 
-exports.waitForPresence = function(selector, elementText, timeout) {
+// elem is a single protractor ElementFinder, such as: `element(by.css('.foo'))`
+// not an ElementArrayFinder, this will not work: `element.all(by.css('.foo'))`
+// example:
+//  waitForElement(element(by.css('.foo')));  // success
+//  waitForElement(element.all(by.css('.foos')));  // fail, incorrect element.all
+var waitForElem = function(elem, timeout) {
+  return browser.wait(EC.presenceOf(elem), timeout || 5000, 'Element not found');
```

Comment:
Good call!  I'll update to `'Element not found: ' + elem.locator().toString()`.  That will print out something like this:
```bash
  - Error: Element not found: by.cssContainingText("a", "STUFF AND THINGS")
```